core/tools/blueos_startup_update/blueos_startup_update

```diff
@@ -147,6 +151,20 @@ def save_file(file_name: str, file_content: str, backup_identifier: str) -> None
     print(run_command(command, False))
 
 
+def file_exists(file_name: str) -> bool:
+    command = f'sudo test -e "{file_name}"'
+    output = run_command(command, True)
+    print(output)
```

Comment:
is these prints for debug ?

Reply:
yes, they allow us to inspect what happened in the Linux side